ui/tests/integration/components/auth-jwt-test.js

```diff
@@ -165,104 +149,80 @@ module('Integration | Component | auth jwt', function (hooks) {
   });
 
   test('oidc: it calls window.open popup window on login', async function (assert) {
+    sinon.replaceGetter(window, 'screen', () => ({ height: 600, width: 500 }));
     await renderIt(this);
-    this.set('selectedAuthPath', 'foo');
+    this.selectedAuthPath = 'foo';
     await component.role('test');
     component.login();
     await waitUntil(() => {
-      return this.openSpy.calledOnce;
+      return this.windowStub.calledOnce;
     });
 
-    cancelTimers();
-    await settled();
-
-    const call = this.openSpy.getCall(0);
+    const call = this.windowStub.lastCall;
     assert.deepEqual(
       call.args,
       ['http://example.com', 'vaultOIDCWindow', 'width=500,height=600,resizable,scrollbars=yes,top=0,left=0'],
       'called with expected args'
     );
+    sinon.restore();
   });
 
-  test('oidc: it calls error handler when popup is closed', async function (assert) {
-    await renderIt(this);
-    this.set('selectedAuthPath', 'foo');
-    await component.role('test');
-    component.login();
-    await waitUntil(() => {
-      return this.openSpy.calledOnce;
-    });
-    this.window.close();
-    await settled();
-    assert.strictEqual(this.error, ERROR_WINDOW_CLOSED, 'calls onError with error string');
-  });
+  // this test technically passes but it's because isTrusted is always false so we do not
+  // even get to the
```

Comment:
```suggestion
  // even get to the error handling logic
```

Reply:
lol yes - this was a WIP commit 🫠 , the comment has changed completely now. 🙈